module/x/gravity/types/genesis.go

```diff
@@ -360,6 +374,22 @@ func validateSlashFractionConflictingEthereumSignature(i interface{}) error {
 	return nil
 }
 
+func validateEthereumBlacklistAddresses(i interface{}) error {
+	strArr, ok := i.([]string)
+	if !ok {
+		return fmt.Errorf("invalid parameter type: %T", i)
+	}
+	for index, value := range strArr {
+		if err := ValidateEthAddress(value); err != nil {
+
+			if !strings.Contains(err.Error(), "empty, index is"+strconv.Itoa(index)) {
```

Comment:
What is this line trying to do, what is the error we are ignoring here?

Reply:
it is to ignore the error on empty string, but I end up removing this line